.travis.yml

```diff
@@ -10,6 +10,15 @@ before_install:
   - sudo mv docker-compose /usr/local/bin
   - docker-compose up -d localstack
   - docker-compose run wait
+  - npm install -g aws-sam-local
+  - sudo apt-get install golang
+  - go get -u github.com/aws/aws-lambda-go/lambda
+  - cd /home/travis/build/ConduitVC/aws-utils/packages/appsync-emulator-serverless/__test__/example/gocomposedjson && go build gocomposedjson.go
+  - cd /home/travis/build/ConduitVC/aws-utils/packages/appsync-emulator-serverless/__test__/example/goemptyjson && go build goemptyjson.go
+  - cd /home/travis/build/ConduitVC/aws-utils/packages/appsync-emulator-serverless/__test__/example/goerror && go build goerror.go
+  - cd /home/travis/build/ConduitVC/aws-utils/packages/appsync-emulator-serverless/__test__/example/gographql && go build gographql.go
+  - cd /home/travis/build/ConduitVC/aws-utils
```

Comment:
@lightsofapollo -- is there a more succinct way of doing this ?

Reply:
```sh
#! /bin/bash -ex

pushd /home/travis/build/ConduitVC/aws-utils/packages/appsync-emulator-serverless/__test__/example/
pushd gocomposedjson
go build gocomposedjson.go
popd
# ...
```